### DISCUSSION TITLE
Changed over my union election data site to quarto, love it

### DISCUSSION THREAD

**Discussion by reuning** [quarto-dev/quarto-cli#8746] [Show and Tell]
### Description

A while ago I built a hacked together site using a bunch of R Scripts and Hugo. I've spent the last few weeks transitioning it over to quarto and am really excited at the ability to edit/update it in a much more useful way.

The site is: https://unionelections.org/

The github for it is here: https://github.com/reuning/unionwebsite every night it runs a github action to update the data. I need to do some cleaning on the action, but I did just get it to work. 

I also put together a quarto project here: https://github.com/reuning/reporting-website that helps to deal with the repeated page nature of the site. It allows me to build a single template page for every state, for example, and then automatically generate each individual state page. 

Thanks for all the work on Quarto.

**Comment by mcanouil** [comment]
Thanks for sharing and glad you liked your experience so far using Quarto CLI.
